CMakeLists.txt

```
@@ -1,92 +1,92 @@
-# 3.9: MPI::MPI_CXX
-# 3.12: CMAKE_PROJECT_VERSION_MAJOR
-# 3.15: PUBLIC_HEADER files for interface libraries
-# 3.23: FILE_SETs for easy export of header-only libraries
-cmake_minimum_required(VERSION 3.23)
+cmake_minimum_required(VERSION 3.25...4.2)
 
 project(
   KokkosComm
+  DESCRIPTION "Experimental communication interfaces for the Kokkos C++ Performance Portability Programming ecosystem"
   LANGUAGES
     CXX
-  VERSION 0.2.0
-  DESCRIPTION "Experimental MPI interfaces (and more!) for the Kokkos C++ Performance Portability Programming ecosystem"
+  VERSION 0.3.0
   HOMEPAGE_URL "https://kokkos.org/kokkos-comm/"
 )
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 include(mpi-vendor)
 
-option(KokkosComm_ENABLE_PERFTESTS "Build KokkosComm perf tests" OFF)
-option(KokkosComm_ENABLE_TESTS "Build KokkosComm tests" OFF)
-option(KokkosComm_ENABLE_MPI "Build KokkosComm with MPI transport" ON)
-option(KokkosComm_ENABLE_NCCL "Build KokkosComm with NCCL transport" OFF)
-option(KokkosComm_ABORT_ON_ERROR "Runtime error checks trigger a global abort" OFF)
-
-# Resolve options
-set(KOKKOSCOMM_ENABLE_PERFTESTS ${KokkosComm_ENABLE_PERFTESTS} CACHE BOOL "" FORCE)
-set(KOKKOSCOMM_ENABLE_TESTS ${KokkosComm_ENABLE_TESTS} CACHE BOOL "" FORCE)
-set(KOKKOSCOMM_ENABLE_MPI ${KokkosComm_ENABLE_MPI} CACHE BOOL "" FORCE)
-set(KOKKOSCOMM_ENABLE_NCCL ${KokkosComm_ENABLE_NCCL} CACHE BOOL "" FORCE)
+# --- OPTIONS --- #
+option(KokkosComm_BUILD_UNITTESTS "Build unit tests. Default: OFF. Values: { ON, OFF }." OFF)
+option(KokkosComm_BUILD_PERFTESTS "Build perf tests. Default: OFF. Values: { ON, OFF }." OFF)
```

```suggestion
option(KokkosComm_BUILD_UNITTESTS "Build unit tests." OFF)
option(KokkosComm_BUILD_PERFTESTS "Build perf tests. " OFF)
```

options are boolean and ccmake or these kind of tools are smart enough to print the default values. It will be easier to maintain to not duplicate this information.